collections/_drafts/agent-forwarding.md

```diff
@@ -140,8 +141,26 @@ keyring into the remote container gets done automatically by the extension.
 
 \[\[endnote\]\]
 
-\`\`\`
-SCP
+:::excerpt{.quote}
```

Comment:
i am stil thinking about how this would like; reference annotations would be nice

```suggestion
:::excerpt{.quote}
```